### PR TITLE
Mansion only totem drops

### PR DESCRIPTION
**Pull request by TheRealRyGuy**
Marked as a draft cause I feel like this is gonna be a pain to test so I haven't tested it yet, the logic looks fine though
Adds #49

### CODE REVIEW

**Review by YouHaveTrouble — CHANGES_REQUESTED** [review]
make sure to include ignoreCancelled and priority in your @ EventHandler annotations

**Comment by TheRealRyGuy** [comment]
I don't want to be ignoring cancelled events here afaik - and isn't `EventPriority#NORMAL` default? seems unnecessary

**Comment by TheRealRyGuy** [comment]
i'm ngl my entire time bukkit coding my brain just inverted what ignoreCancelled meant 

nice

**Comment by TheRealRyGuy** [comment]
still untested but fixed annotations

**Review by YouHaveTrouble — CHANGES_REQUESTED** [review]
Please use `event` isntead of `ev` in listeners for sake of consistency with all other modules.
It's also missing the annotation documenting what it does

**Comment by TheRealRyGuy** [comment]
forgot to comment - this was fixed up

File: src/main/java/org/purpurmc/purpurextras/modules/RaidTotemDropsModule.java [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+import java.util.concurrent.ConcurrentHashMap;
+
+/**
+ * Modifies the drop rate of totems from Evokers spawned in a raid
+ */
+public class RaidTotemDropsModule implements PurpurExtrasModule, Listener {
+
+    private final SplittableRandom random;
+    private final int dropChance;
+    private final Map<UUID, Raider> raiders = new ConcurrentHashMap<>();
```
Comment (YouHaveTrouble) [inline src/main/java/org/purpurmc/purpurextras/modules/RaidTotemDropsModule.java:29]: why are we using concurrent map here?

Comment (TheRealRyGuy) [reply src/main/java/org/purpurmc/purpurextras/modules/RaidTotemDropsModule.java:29]: on the very off chance a raider dies at the same tick a raid starts 
could also just use a sync block but c'est la vie

**Comment by TheRealRyGuy** [comment]
no longer using concurrent hashmap

**Review by YouHaveTrouble — APPROVED** [review]

**Pull request merged**